cvat-core/src/annotations-objects.js

```diff
@@ -1827,20 +1972,35 @@
         }
     }
 
+    class CuboidTrack extends PolyTrack {
+        constructor(data, clientID, color, injection) {
+            super(data, clientID, color, injection);
+            this.shapeType = ObjectShape.CUBOID;
+            for (const shape of Object.values(this.shapes)) {
+                checkNumberOfPoints(this.shapeType, shape.points);
+            }
+        }
+
+        // TODO: interpolation;
```

Comment:
Right now interpolation works based on algorithm for polyshapes?

Reply:
@bsekachev, no i've just adjusted algorithm for rectangles. I guess rectangle and cuboids are more similar.